setup.cfg

```diff
@@ -1,19 +1,48 @@
-[flake8]
-exclude = *.egg-info,.git,.settings,.tox,build,dist,docs,requirements,tmp,*migrations*,*south_migrations*,tests,data
-max-line-length = 119
-
 [metadata]
-license-file = LICENSE
+name = djangocms-multisite
+version = attr: djangocms_multisite.__version__
+url = https://github.com/nephila/djangocms-multisite
+author = Iacopo Spalletti
+author_email = i.spalletti@nephila.digital
+description = django-multisite support for django CMS
+long_description = file: README.rst, HISTORY.rst
+long_description_content_type = text/x-rst
+license = BSD
+license_file = LICENSE
+keywords = djangocms-multisite, django
+classifiers =
+	Development Status :: 5 - Production/Stable
+	Framework :: Django
+	Intended Audience :: Developers
+	License :: OSI Approved :: BSD License
+	Natural Language :: English
+	Framework :: Django
+	Framework :: Django :: 3.2
+	Framework :: Django :: 4.1
+	Framework :: Django :: 4.2
+	Programming Language :: Python :: 3
+	Programming Language :: Python :: 3.9
+	Programming Language :: Python :: 3.10
+	Programming Language :: Python :: 3.11
 
-[wheel]
-universal = 1
+[options]
+include_package_data = True
+install_requires =
+	django-multisite
```

Comment:
this PR requires an upgraded version of django-multisite and we must decide how to go forward

Reply:
@yakky Do you have any update on this? Unfortunately it looks like django-multisite is not being updated anymore.